INSTALL.rst

```diff
@@ -13,28 +48,42 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
     $ workon inspirehep
     (inspirehep)$ cdvirtualenv
     (inspirehep)$ mkdir src
-    (inspirehep)$ git clone https://github.com/inspirehep/inspire-next.git inspirehep
+    (inspirehep)$ git clone https://github.com/inspirehep/inspire-next.git src/inspirehep
 
 
 Install requirements
 ====================
 
-Use `pip` to install all requirements:
+Use `pip` to install all requirements, it's recommended to upgrade pip and
+setuptools to latest too:
 
 .. code-block:: console
 
+    (inspirehep)$ pip install --upgrade pip setuptools
     (inspirehep)$ cdvirtualenv src/inspirehep
     (inspirehep)$ pip install -r requirements.txt --pre --exists-action i
 
+And for development:
+
+.. code-block:: console
+
+    (inspirehep)$ pip install -e .[development]
+
 
 Build assets
 ============
 
 We build assets using `npm`. Make sure you have installed it system wide.
 
+.. note::
+
+    You will have to install a specific version of clean-css until `this issue`_
+    is fixed
+
```

Comment:
Moving the CSS in to the bundle rather than using as an import solves this, so this problem should now be fixed after the changes made yesterday evening.